src/config/local.ts

```diff
@@ -15,6 +15,9 @@ const config: ConfigInterface = {
     debug: true,
     test: false,
   },
+  cors: {
+    origin: 'localhost:3000'
```

Comment:
This one should have the `http` right?

```suggestion
    origin: 'http://localhost:3000'
```